.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,12 +1,15 @@
 # Title
 
-<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines. -->
+<!-- Thanks for sending a pull request! Make sure to follow the contributing guidelines: https://github.com/FreeTubeApp/FreeTube/blob/development/CONTRIBUTING.md -->
 <!-- Important note, we may remove your pull request if you do not use this provided PR template correctly. -->
 
 ## Pull Request Type
 <!-- Please select what type of pull request this is: [x] -->
+<!-- Please remove what is not applicable! -->
 - [ ] Bugfix
 - [ ] Feature Implementation
+- [ ] New Build
+- [ ] I am a New Contributor
```

Comment:
????? (for `Pull Request Type`)

Reply:
Yeah i didnt know where to put this one. Preston mentioned new contributors in the last release notes https://github.com/FreeTubeApp/FreeTube/releases/tag/v0.18.0-beta. Let me know where it should go

Reply:
Definitely not `Pull Request Type` XD
If no idea add new section `Others` (well I have no idea about the title)

Reply:
What about moving it to the top?
https://github.com/FreeTubeApp/Test-Repo/pull/120

Reply:
Looks reasonable 
Better than `Pull Request Type` anyway 😏

Reply:
When you let GitHub automatically generate the release notes, it generates the new contributors section automatically, but as you are doing it manually now, you'll probably have to manually generate that list too.

Reply:
@absidue do you think there is maybe a better place for it in the template?

Reply:
@PikachuEXE does the following look better then moving it to the top in FreeTubeApp/Test-Repo#120?

![hidden](https://user-images.githubusercontent.com/73130443/200387692-15d6e843-ed6f-47c3-bcaf-e3196fd86196.PNG)

Reply:
Can we generate both?
GH generated one and our generated one
Then merge manually

@efb4f5ff-1298-471a-8973-3d47447115dc 
That's too hidden :S 
Just add to the top

Reply:
> Can we generate both? GH generated one and our generated one Then merge manually.

So there is a auto generate button when i edit the release draft. It didnt do a thing when i clicked in it so i think its not possible.

Reply:
GitHub won't overwrite the draft. It will only generate stuff if the release is empty.

Reply:
Ah okay.

> We could also opt for not putting the keywords in the PR body and monitor ourselves if somebody is new. If somebody is new we apply label manually 

Do you think that this is a better solution then leaving it up to the author of the PR?

Reply:
@efb4f5ff-1298-471a-8973-3d47447115dc What's your solution for PRs that shouldn't be in release notes? e.g. users don't care about code refactoring, so none of those PRs of mine should end up in the release notes.

I guess my question is, is it worth having this release drafter stuff, when Preston is going to have to go through and clean it all up (remove unnecessary ones, reword the messages and probably recategorise a bunch of them) anyway?

Reply:
> @efb4f5ff-1298-471a-8973-3d47447115dc What's your solution for PRs that shouldn't be in release notes? e.g. users don't care about code refactoring, so none of those PRs of mine should end up in the release notes.

PR's like that will likely end up in the 'merged without label' bucket as seen in https://github.com/FreeTubeApp/Test-Repo/releases/tag/untagged-e0442ce3c8a15579f96b so Preston can deceide what to do with them.

> I guess my question is, is it worth having this release drafter stuff, when Preston is going to have to go through and clean it all up (remove unnecessary ones, reword the messages and probably recategorise a bunch of them) anyway?

Have thinking about this allot about this lately and i think that with this workflow he at least doesn't have to create the headers and categorize allot of PR's because they are already put in the right category by the authors of the PRs. 

The only things he needs to do is reword but he also needed to do that before so nothing changes there and check the PR's in the bucket without label.

Created this so he has less to do when we have to do a release but ultimately it is his choice if this is going to be implemented or not.

Reply:
Is this PR now good enough to ask for Preston's opinion?
If so just ask in chat or something

Reply:
Almost done

Need help with
- https://github.com/FreeTubeApp/FreeTube/pull/2825#discussion_r1015750338
- Address the stuff in additional context header located in pr body